### PR TITLE
Secrets panel logging is better

### PR DESCRIPTION
**Pull request by ChangelingRain**
it's still the secrets panel though so like.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Would be a good PR to take a hack at https://github.com/tgstation/tgstation/issues/26347 in

**Comment by ChangelingRain** [comment]
absolutely not

**Comment by PKPenguin321** [comment]
Nevermind

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -198,38 +199,35 @@
 			if(!check_rights(R_ADMIN))
 				return
 			feedback_inc("admin_secrets_fun_used",1)
```
Comment (lzimann) [inline code/modules/admin/secrets.dm]: Forgot to remove this one.

**Pull request merged**